src/goto-checker/solver_factory.h

```diff
@@ -1,16 +1,16 @@
 /*******************************************************************\
 
-Module: Bounded Model Checking for ANSI-C + HDL
+Module: Solver Factory
 
-Author: Daniel Kroening, kroening@kroening.com
+Author: Daniel Kroening, Peter Schrammel
 
 \*******************************************************************/
 
 /// \file
-/// Bounded Model Checking for ANSI-C + HDL
+/// Solver Factory
 
-#ifndef CPROVER_CBMC_CBMC_SOLVERS_H
-#define CPROVER_CBMC_CBMC_SOLVERS_H
+#ifndef CPROVER_GOTO_CHECKER_SOLVER_FACTORY_H
+#define CPROVER_GOTO_CHECKER_SOLVER_FACTORY_H
```

Comment:
Wishlist, possibly to be done in a separate PR: could a lot of includes be purged from the header file and the implementation of several methods be moved to the .cpp file? Really nothing needs to be inlined for performance reasons. With such a move we would eventually end up with a clean and readable _interface_.

Reply:
https://github.com/diffblue/cbmc/pull/3588